static/styles/rendered_markdown.css

```diff
@@ -350,8 +357,9 @@
     .twitter-image img,
     .message_inline_image img,
     .message_inline_ref img {
-        height: auto;
-        max-height: 100%;
+        height: 100%;
+        width: 100%;
+        object-fit: cover;
```

Comment:
Or, we could do it the no-crop way:

![image](https://user-images.githubusercontent.com/72784348/153057655-73945007-905a-4be4-be2b-b91cf2a2a0ec.png)

vs

![image](https://user-images.githubusercontent.com/72784348/153057965-7e338d65-6f8b-4a38-ad26-dbfefeef3146.png)

Reply:
I think this is a good question to post in the #design thread for discussion; I'm not sure what's best off the top of my head.

Reply:
I would prefer `no-crop` since a cropped image can give wrong context sometimes.

Reply:
> I think this is a good question to post in the #design thread for discussion; I'm not sure what's best off the top of my head.

I've posted the question in the original topic for the parent issue. I'm still getting familiar with Zulip etiquette and how to best things so please let me know if a new topic would be better.

Reply:
> I would prefer `no-crop` since a cropped image can give wrong context sometimes. It could be problematic for an image with extreme dimensions, though.

I agree on all counts! I have a bias for `no-crop` myself since cropping can make things very unrepresentative and poor looking especially with highly compressed images (looking at you WhatsApp 👀 )

Reply:
We should use [`object-fit: scale-down`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit#scale-down). This shrinks large images so they fit in both dimensions (so the image is not cropped), but without enlarging small images.

Reply:
Indeed! I think the behavior this yields would be optimal